psyslog.js

```diff
@@ -23,21 +51,9 @@ if (args.config) {
   } catch (e) {
     process.stderr.write(`could not load settings file, using defaults: ${e.message}`)
   }
+  options = Object.assign({}, options, userOptions)
```

Comment:
The more I think about, the more I think we should go with this from the beginning:

```suggestion
  options = Object.assign({}, userOptions, options)
```

Then update the docs accordingly to relate the order of precedence. I'd also rename `userOptions` to `jsonOptions`.

Reply:
With precedence you mean option flags overwrite json options?